### DISCUSSION TITLE
Tests have a circular dependency

### DISCUSSION THREAD

**Discussion by jgarte** [pygments/pygments#2587] [Q&A]
Hi,

It looks like both the Nix and Guix packages mention pygments as having a circular dependency and disable there tests:

https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/development/python-modules/pygments/default.nix#L25

https://git.savannah.gnu.org/cgit/guix.git/tree/gnu/packages/python-xyz.scm?h=master#n6126

Do you have any suggestions for running the test suite, given this issue?

**Comment by Anteru** [comment]
Is that "tests require sphinx"? They certainly don't do that. I'm puzzled -- can you elaborate what the circular dependency is? Clearly we depend on Sphinx for documentation, which depends on Pygments, but that's the nature of being a part of Sphinx. That doesn't mean it's actually a problem as we install a different version of Pygments during the documentation build.

**Reply by jeanas** [reply]
I think the problem is that pytest requires Pygments.

@jgarte Sorry, but there is not much we can do about that.
